include/fire/Event.h

```diff
@@ -506,7 +560,13 @@ class Event {
    * structure to hold event data in memory
    */
   struct EventObject {
-    /// the data for save/load
+    /**
+     * the data for save/load
+     *
+     * this pointer will be nullptr for the data objects
+     * that have been seen in an input file but not requested
+     * by a Processor through Event::get
```

Comment:
This comment was put in when I was implementing a different idea. It is no longer true. The `EventObject` only exists when an in-memory object exists.

```suggestion
```